tests/wicked/aggregate/sut/t17_teaming_ab_ethtool.pm

```diff
@@ -0,0 +1,37 @@
+# SUSE's openQA tests
+#
+# Copyright © 2019 SUSE LLC
+#
+# Copying and distribution of this file, with or without modification,
+# are permitted in any medium without royalty provided the copyright
+# notice and this notice are preserved.  This file is offered as-is,
+# without any warranty.
+
+# Summary: Teaming, Active-Backup Ethtool
+# Maintainer: Anton Smorodskyi <asmorodskyi@suse.com>
+#             Jose Lausuch <jalausuch@suse.com>
+#             Clemens Famulla-Conrad <cfamullaconrad@suse.de>
+
+use Mojo::Base 'wickedbase';
+use testapi;
+
+
+sub run {
+    my ($self, $ctx) = @_;
+    record_info('INFO', 'Teaming, Active-Backup Ethtool');
+    $self->setup_team('ab-ethtool', $ctx->iface(), $ctx->iface2());
+    $self->validate_interfaces('team0', $ctx->iface(), $ctx->iface2(), 0);
```

Comment:
why you don't want to ping other side here ?

Reply:
the test suite doesn't require so at this step, only at the end:
https://gitlab.suse.de/wicked-maintainers/wicked-testsuite/blob/stable/features/wicked_9_aggregation.feature#L135

Reply:
ok